src/commonMain/kotlin/io/github/koalaplot/core/util/HoverableElementArea.kt

```diff
@@ -184,4 +192,8 @@ private class HoverableElementAreaScopeImpl(private val sender: Sender) :
                 }
             }
     }
+
+    override fun getCurrentPointer(): Offset {
```

Comment:
When the mouse is moved the currentPosition is updated, but how will client code know it needs to call ```getCurrentPosition``` to update whatever it wants to do with an updated mouse position?